python/tskit/trees.py

```diff
@@ -833,6 +833,57 @@ def unrank(rank, num_leaves):
         """
         return combinatorics.RankTree.unrank(rank, num_leaves).to_tsk_tree()
 
+    def count_topologies(self, sample_sets=None):
+        """
+        Calculates the distribution of embedded topologies for every combination
+        of the sample sets in ``sample_sets``. ``sample_sets`` defaults to all
+        samples in the tree grouped by population. In this case, the topologies
+        produced represent evolutionary topologies of the populations in this
+        tree.
```

Comment:
I think you can delete this last sentence - I don't think it's adding anything. If you keep it, don't use "evolutionary" - there's nothing about evolution here - perhaps you mean "genealogical"?